### DISCUSSION TITLE
Notifications counter visible when repo is DMCA-ed

### DISCUSSION THREAD

**Discussion by SoapDev2018** [community/community#108237] [Repositories]
### Select Topic Area

Bug

### Feature Area

Issues

### Body

I am encountering a GitHub bug. I will try to explain the scenario and ways to reproduce this issue:
1. Have a repository (which you do not own) and ensure that it has sent some notifications (about releases or something else).
2. The repository gets targeted by DMCA and gets disabled.
3. Go back to your GitHub notifications; the notifications from this disabed repository still show up. Clicking on them will lead to nowhere since the repository has already been disabled.

This issue is visible both on GitHub Mobile and GitHub Web.

I will paste supporting screenshots showing the same.
![image](https://github.com/community/community/assets/36785676/046f4d19-701c-4208-9157-4dd6f6773536)
![image](https://github.com/community/community/assets/36785676/fc22ca11-9afb-41ae-a548-2662fc5d97b9)

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @SoapDev2018 ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Projects and Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!

**Comment by Osyotr** [comment]
This issue is really annoying. Someone mentioned me in some gist which got removed and now the notification badge stuck
![image](https://github.com/community/community/assets/8740768/a66daa4f-a012-4aee-a5f6-438622035651)

**Comment by StringKe** [comment]
The same problem exists, there are always a few unread messages present and I can't check it.

**Comment by Vasfed** [comment]
Managed to reset notifications counter via api request:
```sh
gh api notifications -X PUT -f last_read_at='2024-03-10T00:00:00Z' -f read='true'
```
(`gh` is the github official cli utility, you need to be authenticated, also possible via `curl`, but have to handle token and full api uri manually)

**Comment by StevenSYS** [comment]
Here's the curl version of the command:
```bash
curl -L \
	-X PUT \
	-H "Accept: application/vnd.github+json" \
	-H "Authorization: Bearer YOUR_TOKEN_GOES_HERE" \
	-H "X-GitHub-Api-Version: 2026-03-10" \
	"https://api.github.com/notifications" \
	-d "{ \"last_read_at\": \"2026-05-22T00:00:00z\", \"read\": true }"
```

Make sure to replace `YOUR_TOKEN_GOES_HERE` with a token you generated that has the notification scope enabled.
